.openpublishing.redirection.json

```diff
@@ -1904,6 +1904,14 @@
             "source_path": "docs/csharp/programming-guide/classes-and-structs/classes.md",
             "redirect_url": "/dotnet/csharp/fundamentals/types/classes"
         },
+	{
+            "source_path": "docs/csharp/programming-guide/classes-and-structs/finalizers.md",
+            "redirect_url": "/dotnet/csharp/programming-guide/classes-and-structs/destructors"
+        },
+        {
+            "source_path": "docs/csharp/programming-guide/classes-and-structs/finalizers.md",
+            "redirect_url": "/docs/csharp/programming-guide/classes-and-structs/destructors.md"
+        },
```

Comment:
These lines should be removed (it looks like they were added twice)

Reply:
I removed it but there is still some error. As I suggested before but obviously not enough clearly.